shared/test/test_sound_library_api.rb

```diff
@@ -0,0 +1,23 @@
+require_relative 'test_helper'
+require 'sound_library_api'
+
+class SoundLibraryTest < Minitest::Test
+  include Rack::Test::Methods
+  include SetupTest
+
+  def build_rack_mock_session
+    @session = Rack::MockSession.new(SoundLibraryApi, 'studio.code.org')
+  end
+
+  def test_get_sound
+    stubs(:cache_for).returns(nil)
+    get '/api/v1/sound-library/test_sound.mp3'
+    assert last_response.ok?
+  end
+
+  def test_get_deleted_sound
+    stubs(:cache_for).returns(nil)
+    get '/api/v1/sound-library/test_sound.mp3'
+    assert last_response.ok?
```

Comment:
The other thing I worry about is that once we have VCR in play (and this is somewhat more of a general problem and not necessarily specific to the test), I'm not sure this test ever fails. Rather than testing "does get sound work" it ends up testing "did get sound work this one time when I recorded the VCR". Not sure I have solutions :-P